RoaringBitmap/src/main/java/org/roaringbitmap/buffer/MappeableArrayContainer.java

```diff
@@ -246,6 +247,26 @@ public boolean isFull() {
     return false;
   }
 
+  @Override
+  public void orInto(long[] bits) {
+    for (int i = 0; i < this.getCardinality(); ++i) {
+      char value = content.get(i);
+      bits[value >>> 6] |= (1L << value);
+    }
+  }
+
+  @Override
+  public void andInto(long[] bits) {
+    int prev = 0;
```

Comment:
This is probably correct and good but it made me cry a little. I can't think of a better way, so ok.

Reply:
There’s a choice between allocating lots of tiny arrays and doing extra work in the same 8kb array over an over again. In Java, limiting allocation wins and peak performance is off the table to begin with.

Reply:
I realize that. I did spend 5 minutes or so trying to think of a better way and I gave up.

Reply:
I think you're right, it just didn't occur to me. Let me know if you think of something.